include/GafferUI/StandardConnectionGadget.h

```diff
@@ -92,6 +92,11 @@ class StandardConnectionGadget : public ConnectionGadget
 
 		bool updateUserColor();
 
+		bool __dropDot( Imath::V2f position );
+		void __updateDotPreviewBounds( const ButtonEvent &event );
+		bool __keyPressed( const KeyEvent &event );
```

Comment:
There's no need for the "__" prefix in C++ - in fact I don't think it's even allowed. In C++ we have proper access control via `private :` and `protected :`, and use that instead. It's only in Python that we're in the wild west and have to use a naming convention to denote protected/private.